### PR TITLE
Create bipartite.py

### PR DESCRIPTION
**Pull request by jntushar**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by jntushar** [comment]
Implemented Bipertite graph

File: graphs/bipartite.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Uses python3
+import sys
+
+
+def check(partition, adj):
```
Comment (cclauss) [inline graphs/bipartite.py:5]: Type hints and doctests as discussed in CONTRIBUTING.md?  Could we have a more self documenting function name?  What exactly are we checking?

File: graphs/bipartite.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                if partition[i] == 'W':
+                    return 0
+            else:
+                if partition[i] == 'B':
+                    return 0
+        test += 1
+    return 1
+
+
+def bipartite(adj):
```
Comment (cclauss) [inline graphs/bipartite.py:20]: Type hints and doctests as discussed in CONTRIBUTING.md?

File: graphs/bipartite.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                    partition[i] = 'B'
+                else:
+                    partition[i] = 'W'
+        if flag == 1:
+            step += 1
+    return check(partition, adj)
+
+
+if __name__ == '__main__':
+    input = sys.stdin.read()
```
Comment (cclauss) [inline graphs/bipartite.py:43]: `input()` is a builtin in Python so that should not be used as a variable name.

File: graphs/bipartite.py (hunk 1 above)
Comment (jntushar) [reply graphs/bipartite.py]: updated the function name

**Pull request closed without merging**